Package.swift

```diff
@@ -42,6 +48,10 @@ let package = Package(
             name: "Auth",
```

Comment:
let's create issue to rename Auth and Sign to WalletConnectAuth and WalletConnectSign?

Reply:
Goot point